File: web/model/customer.js
function Customer() {
   var __id ;
   var __name ;
   var __address ;
   var __tel ;

   this.setId = function (id) {
      __id = id;
   }

   this.getId = function () {
       return __id;
   }

   this.setName = function (name) {
        __name = name;
   }

   this.getName = function () {
        return __name;
   }

   this.setAddress = function (address) {
     __address = address;
   }

   this.getAddress = function () {
        return __address;
   }

   this.setTel = function (tel) {
      __tel = tel;
   }
   this.getTel = function () {
        return __tel;
   }
}

File: src/model/Customer.java
package model;
/*
author :Himal
version : 0.0.1
*/

public class Customer {
   private String nic;
   private String name;
   private String address;
   private int tel;

    public Customer() {
    }

    public Customer(String nic, String name, String address, int tel) {
        this.nic = nic;
        this.name = name;
        this.address = address;
        this.tel = tel;
    }



    public String getNic() {
        return nic;
    }

    public void setNic(String nic) {
        this.nic = nic;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public int getTel() {
        return tel;
    }

    public void setTel(int tel) {
        this.tel = tel;
    }
}
